deep_qa/data/tokenizers/word_and_character_tokenizer.py

```diff
@@ -81,10 +81,10 @@ def embed_input(self,
         # mask_split_axis argument to VectorMatrixSplit.
         words, characters = VectorMatrixSplit(split_axis=-1)(input_layer)
         word_embedding = embed_function(words,
-                                        embedding_name='word_' + embedding_name,
```

Comment:
This is changing the meaning of that parameter: previously `embedding_name` was used as the layer name, and the key in a dictionary that stored actual embedding layers.  Now, `embedding_name` is used as the key to the _parameters_ that you pass.  If by default you have `_embeddings` as a suffix to the embedding name, that means that in your parameter file you need to have something like this:

```json
"embeddings": {
  "word_embeddings": {
    "dimension": 100
  },
  "character_embeddings": {
    "dimension": 8
  }
}
```

That felt a bit redundant to me.  Instead, it looks like this:
```json
"embeddings": {
  "words": {
    "dimension": 100
  },
  "characters": {
    "dimension": 8
  }
}
```

and if for some reason you want to have multiple embeddings of words, you can specify them like this:
```json
"embeddings": {
  "words_A0": {
    "dimension": 100
  },
  "words_A1": {
    "dimension": 100
  }
}
```

The layer itself still has `_embedding` appended to it, so you still see something like `words_embedding` in the model summary.